packages/next-app/src/app/internalmanager/page.tsx

```diff
@@ -92,7 +95,72 @@ export default function Page() {
     }
   };
 
+  async function getInternalBalances() {
+    if (!addressLower) return;
+
+    const { user } = await pools
+      .gql(chain?.id.toString() || "1")
+      .InternalBalance({
+        userAddress: addressLower as `0x${string}`,
+      });
+
+    if (user?.userInternalBalances) {
+      const tokensWithBalanceRaw = user.userInternalBalances.filter(
+        (token) => token.balance > 0
+      );
+
+      const incompletedTokens: { token: string; balance: number }[] = [];
+      const alreadyCompletedTokens: IToken[] = [];
+
+      tokensWithBalanceRaw.forEach((token) => {
+        if (token.token in tokenDictionary) {
```

Comment:
`#comment` that is, if it's a key of `tokenDictionary`

Reply:
this part of the code was removed with the new subgraph implementation